authz/openfga/v2/changed.diff

```diff
@@ -1,18 +0,0 @@
---- authz/openfga/v1/schema.fga	2024-12-09 12:10:28
```

Comment:
why get rid of this file? I thought it was nice to see the changes like this in PRs etc

Reply:
The principal is now a bit different:
All backwards compatible changes can be made right there in the `store.fga.yaml` and the folder version increased.
Increasing the folder version is just a failsafe to prevent us from not updating rust.

Only for breaking changes, that require tuple migrations, we will create a new folder.
Thus, for backwards compatible changes we can now simply use the git history to get changes.
Happy to add the diff file for new majors.